test/test_device.cpp

```
@@ -0,0 +1,758 @@
+                nullptr, nullptr),
+            QDMI_ERROR_INVALIDARGUMENT);
+  EXPECT_THAT(
+      AMAZON_BRAKET_QDMI_device_session_query_device_property(
+          session,
+          static_cast<QDMI_Device_Property>(QDMI_DEVICE_PROPERTY_COUPLINGMAP),
+          0, nullptr, nullptr),
+      testing::AnyOf(QDMI_SUCCESS, QDMI_ERROR_NOTSUPPORTED));
+  AMAZON_BRAKET_QDMI_device_session_free(uninitializedSession);
+}
+
+TEST_F(AmazonBraketQDMISpecificationTest, AwsSpecificDeviceProperties) {
```

Clang (14.0.6)

[warning] invalid case style for function 'TEST_F'

(readability-identifier-naming)